cmd/ocsp-updater/main.go

```diff
@@ -354,7 +367,12 @@ func (updater *OCSPUpdater) findRevokedCertificatesToUpdate(batchSize int) ([]co
 	var statuses []core.CertificateStatus
 	_, err := updater.dbMap.Select(
 		&statuses,
-		`SELECT * FROM certificateStatus
+		`SELECT
```

Comment:
@jsha Correct! Should I call that out in the PR desc or put a comment here explaining it? I found that I'd apply the migration and if there was an ocsp-updater running in the docker container it would start to throw GORP errors from the `select *` pulling in an unknown field.

_(I also think we should always avoid `SELECT *` as a general rule)_